zest/releaser/pypi.py

```diff
@@ -176,26 +165,18 @@ def reload(self):
         upload fails, you edit the .pypirc file to fix the account
         settings, and tell release to retry the command.
         """
-        self._read_configfile(use_setup_cfg=self.use_setup_cfg)
-
-    def _read_configfile(self, use_setup_cfg=True):
-        """Read the PyPI config file and store it (when valid).
+        self._read_configfile()
+    
+    def zest_releaser_config(self):
+        return extract_zestreleaser_configparser(self.config, self.config_filename)
 
-        Usually read the setup.cfg too.
-        """
-        rc = self.config_filename
-        if not os.path.isabs(rc):
-            rc = os.path.join(os.path.expanduser("~"), self.config_filename)
```

Comment:
This part should not have been removed.  It means `.pypirc` is not found, where previously we would have checked `~/.pypirc`.  See issue #420.
I will check if putting these lines back works.

Reply:
Yes, that does the trick.  I am preparing a PR.